docs/advanced/secure-setup.md

```diff
@@ -0,0 +1,372 @@
+    photograph.
+    A digital copy of the hash could be manipulated by any malicious program
+    running on the system.
+
+### Setting up Wallet-B
+
+Install an operating system you feel comfortable with.
+Ubuntu desktop is a good choice for beginners (there is no need for a GUI).
+
+**On a general system, you can install the OS using a Linux ISO written to a USB. I recommend [unetbootin](https://unetbootin.github.io/)**
+
+??? note "Raspberry Pi specific (click to expand)"
+
+    <https://www.raspberrypi.org/software/> has rpi-imager which has a nice easy to use interface and has Ubuntu Desktop listed under `Other general purpose OS` -> `Ubuntu` -> `Ubuntu Desktop`.
+
+    Once rpi-imager is done, the disk/card with the OS installed should mount volumes.
+
+    We will need to work with
+
+    ```no-highlight
+    writeable
+    *boot
+    ```
+
+    Edit the config.txt file in *boot
+
+    Find the text [all]
+
+    and add the following lines below it:
+
+    ```no-highlight
+    dtoverlay=disable-wifi
+    dtoverlay=disable-bt
+    ```
+
+    This will disable the WiFi and Bluetooth from being initialized when the system boots. 
+
+You can now copy over `~/copytob.tar.gz` to a writeable volume.
+
+For ease of use, let us make a folder called `bconfig` and copy it over.
+
+(You might need to use an admin or sudo permissions)
+
+eg (These will be different for LiveUSB's)
+`sudo mkdir /media/{yourusername}/writable/bconfig`
+`sudo cp ~/copytob.tar.gz /media/{yourusername}/writable/bconfig`
+
+Now unmount the disk from your system.
+**This will be the last time you will ever connect this disk to a system with internet access.**
+
+Now boot `Wallet-B` with the disk inserted **(DO NOT CONNECT YOUR ETHERNET CABLE)**.
+Depending on the OS it should show you a system setup and a default user creation menu, proceed with a strong password.
+Then once the system is installed it should reboot.
+
+Once logged in get familiar with the system, adjust the clock, etc, and then open a terminal.
+
+??? note "Raspberry Pi specific (click to expand)"
+
+    Optionally you may choose to disable WiFi and Bluetooth at the kernel level too. 
+
+    Create a file in
+
+    ```no-highlight
+    /etc/modprobe.d/raspi-blacklist.conf
+    ```
+
+    and add the contents
+
+    ```no-highlight
+    blacklist brcmfmac
+    blacklist brcmutil
+    blacklist hci_uart
+    blacklist btbcm
+    blacklist btintel
+    blacklist rfcom
+    blacklist btqca
+    blacklist btsdio
+    blacklist bluetooth
+    ```
+
+Disable WiFi and Bluetooth using rfkill:
+
+```no-highlight
+sudo rfkill block wifi
+sudo rfkill block bluetooth
+```
+
+Now to setup the ufw rules run the following commands.
+
+```no-highlight
+sudo ufw default deny outgoing
+sudo ufw default deny incoming
+sudo ufw allow out to {LOCAL_IP_OF_COMPUTER-A} port 9109
+sudo ufw enable
+```
+
+Replace {LOCAL_IP_OF_COMPUTER-A} with the IP address of `Computer-A`
+
```

Comment:
It would be especially good to have an example to compare against that even included a dumb, fake IP address like `192.168.1.50`, assuming that the IP of Computer A is that, that way we don't get people trying to enter commands like this:

`sudo ufw allow out to {192.168.1.50} port 9109`

and then asking why it didn't work